ckanext/datastore/db.py

```diff
@@ -102,7 +102,9 @@ def _get_engine(data_dict):
     engine = _engines.get(connection_url)
 
     if not engine:
-        engine = sqlalchemy.create_engine(connection_url)
+        import pylons
+        extras = {'url': connection_url}
+        engine = sqlalchemy.engine_from_config(pylons.config, 'ckan.datastore.sqlalchemy.', **extras)
```

Comment:
This is causing the pep8 tests to fail :( 

Probably wants to be 
```
        engine = sqlalchemy.engine_from_config(pylons.config, 
                                               'ckan.datastore.sqlalchemy.', 
                                                **extras)
```

but I can't push to your branch to fix it. Sorry.